src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -25,33 +28,45 @@
         return internshipService.getAll();
     }
 
+    @AdminAccessControl
```

Comment:
This endpoint should be open for non-authorized users, doesn't it?

Reply:
Fixed. I don't know why i set permission